integration-tests/test/ovmcontext.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { ethers } from 'hardhat'
+import { injectL2Context } from '@eth-optimism/core-utils'
+import { expect } from 'chai'
+import { sleep, l2Provider, l1Provider, getAddressManager } from './shared/utils'
+import { OptimismEnv } from './shared/env'
+import { getContractFactory } from '@eth-optimism/contracts'
+import { Contract, ContractFactory, Wallet, BigNumber } from 'ethers'
+
+/**
+ * These tests cover the OVM execution contexts. In the OVM execution
+ * of a L1 to L2 transaction, both `block.number` and `block.timestamp`
+ * must be equal to the blocknumber/timestamp of the L1 transaction.
+ */
+describe.only('OVM Context: Layer 2 EVM Context', () => {
```

Comment:
```suggestion
describe('OVM Context: Layer 2 EVM Context', () => {
```

Reply:
Sorry, accidentally committed that. Rebased it out